mpt_trie/src/trie_subsets.rs

```diff
@@ -21,13 +22,15 @@ use crate::{
 /// The output type of trie_subset operations.
 pub type SubsetTrieResult<T> = Result<T, SubsetTrieError>;
 
-/// Errors that may occur when creating a subset [`PartialTrie`].
+/// We can't construct the error until we unwind the stack from the error
```

Comment:
I don't really understand what this means. 
Stack unwinding is the rust runtime throwing a `panic`, running destructors on each stack frame until it either hits main, or a `catch_unwind`. 
But I don't see any panic or panic handling code here (and I'd object if there were).

Reply:
Yeah, maybe that was the wrong term to use here. I really just meant we need to wait until we go back up the call stack until we get back to the point where the recursive calls start. I'll reword this.